web/themes/custom/kitrab/src/scss/_breackpoints.scss

```
@@ -0,0 +1,11 @@
+$xs:200px;
+$sm:400px;
+$md:768px;
+$lg:992px;
+$xl:1200px;
+@mixin md (){
+  @media (max-width: $md) {
+    @content;
+  }
+}
```

We need to create mixins for some general cases.
```suggestion
@mixin media-breakpoint-max ($size){
  @media all and (max-width: $size) {
    @content;
  }
}
@mixin media-breakpoint-min ($size){
  @media all and (min-width: $size) {
    @content;
  }
}
```